test/e2e/networking.go

```diff
@@ -83,7 +83,6 @@ var _ = framework.KubeDescribe("Networking", func() {
 
 		By("checking kube-proxy URLs")
 		config.GetSelfURL(ports.ProxyHealthzPort, "/healthz", "200 OK")
-		config.GetSelfURL(ports.ProxyStatusPort, "/proxyMode", "iptables") // the default
```

Comment:
We definitely shouldn't just remove it. We should come up with a better way to check it, but removing it is not a solution.